### ISSUE TITLE
UnicodeDecodeError  0xe6  0x9c

### ISSUE THREAD

**Issue by gn02394847** [PokemonGoF/PokemonGo-Bot#2677]
2016-08-07 14:34:29,445 [       cli] [INFO] Most Perfect Pokemon: 
Traceback (most recent call last):
  File "pokecli.py", line 493, in <module>
    main()
  File "pokecli.py", line 129, in main
    raise e
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 0: ordinal not in range(128)
2016-08-07 14:34:29,452 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)
Traceback (most recent call last):

**Comment by SGColdSun** [comment]
Likely fixed in https://github.com/PokemonGoF/PokemonGo-Bot/issues/2673

**Comment by pmquan** [comment]
Temporary disable "health_record": false in your config file may help

**Comment by gn02394847** [comment]
2016-08-07 14:32:48,605 [MoveToFort] [INFO] [moving_to_lured_fort] Moving towards pokestop XXX - 0.05km (attraction of lure 0.05km)
2016-08-07 14:32:54,949 [MoveToFort] [INFO] [moving_to_lured_fort] Moving towards pokestop XXX - 0.04km (attraction of lure 0.04km)
2016-08-07 14:33:01,442 [       cli] [INFO] 
2016-08-07 14:33:01,442 [       cli] [INFO] Ran for 0:04:32
2016-08-07 14:33:01,442 [       cli] [INFO] Total XP Earned: 760  Average: 10049.83/h
2016-08-07 14:33:01,442 [       cli] [INFO] Travelled 0.01km
2016-08-07 14:33:01,442 [       cli] [INFO] Visited 1 stops
2016-08-07 14:33:01,442 [       cli] [INFO] Encountered 1 pokemon, 1 caught, 0 released, 0 evolved, 1 never seen before
2016-08-07 14:33:01,442 [       cli] [INFO] Threw 1 pokeball
2016-08-07 14:33:01,442 [       cli] [INFO] Earned 100 Stardust
2016-08-07 14:33:01,442 [       cli] [INFO] 
2016-08-07 14:33:01,442 [       cli] [INFO] Highest CP Pokemon: Horsea [CP: 72] [IV: 11/10/4] Potential: 0.56 
2016-08-07 14:33:01,442 [       cli] [INFO] Most Perfect Pokemon: Horsea [CP: 72] [IV: 11/10/4] Potential: 0.56 
Traceback (most recent call last):
  File "pokecli.py", line 493, in <module>
    main()
  File "pokecli.py", line 129, in main
    raise e
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 0: ordinal not in range(128)
2016-08-07 14:33:01,450 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)
Traceback (most recent call last):

**Comment by chinkong83** [comment]
@pmquan it's not help, but thanks

**Comment by pmquan** [comment]
I found there was something wrong with catch_lured_pokemon function. So I disabled it temporarily.

**Comment by pmquan** [comment]
For any developers came here:

> Error in pokecli.py on line 86
> Error in /root/PokemonGo-Bot/pokemongo_bot/__init__.py on line 393
> Error in /root/PokemonGo-Bot/pokemongo_bot/cell_workers/catch_lured_pokemon.py on line 11
> Error in /root/PokemonGo-Bot/pokemongo_bot/cell_workers/catch_lured_pokemon.py on line 41
> Error in /root/PokemonGo-Bot/pokemongo_bot/cell_workers/base_task.py on line 26
> Error in /root/PokemonGo-Bot/pokemongo_bot/event_manager.py on line 61
> Traceback (most recent call last):
>   File "pokecli.py", line 496, in <module>
>     main()
>   File "pokecli.py", line 132, in main
>     raise e
> UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)

I think the error happened when the fort_name included the Unicode character. 

The temporary solution: Edit the file pokemongo_bot/cell_workers/catch_lured_pokemon.py
Then comment out those lines:
```

             #self.emit_event(
            #    'lured_pokemon_found',
            #    formatted='Lured pokemon at fort {fort_name} ({fort_id})',
            #    data=result
            #)
```

**Comment by chinkong83** [comment]
@pmquan thanks a lot

**Comment by leadboots5** [comment]
should be fixed with #2720 otherwise open a new issue for fresh eyes on the current code.
